mcs/tools/xbuild/frameworks/net_4.6.2.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FileList  Name=".NET Framework 4.6.2" TargetFrameworkDirectory="..\..\..\..\4.5">
+<FileList  Name=".NET Framework 4.6.2" TargetFrameworkDirectory="..\..\..\..\4.6.2-api">
```

Comment:
No we don't since MSBuild is looking into the xbuild-frameworks directory already: https://github.com/Microsoft/msbuild/blob/2634f05a660b14c64ae4e924d900dd200d5032e7/src/Shared/FrameworkLocationHelper.cs#L871